dot/sync/chain_sync.go

```diff
@@ -237,18 +238,14 @@ func (cs *chainSync) stop() error {
 	}
 }
 
-func (cs *chainSync) isBootstrap() (bestBlockHeader *types.Header, syncTarget uint,
+func (cs *chainSync) currentSyncInformations() (bestBlockHeader *types.Header, syncTarget uint,
```

Comment:
```suggestion
func (cs *chainSync) currentSyncInformation() (bestBlockHeader *types.Header, syncTarget uint,
```

Reply:
I will keep it as it is since this function is returning more than one data, so makes sense to keep it in plural